coreneuron/nrniv/main1.cpp

```diff
@@ -518,7 +522,14 @@ extern "C" int run_solve_core(int argc, char** argv) {
         double min_report_dt = INT_MAX;
         int report_buffer_size = nrnopt_get_int("--report-buffer-size");
         for (size_t i = 0; i < configs.size(); i++) {
-            register_report(dt, tstop, delay, configs[i]);
+            std::shared_ptr<ReportHandler> report_handler;
+            if(0 == strcmp( configs[i].format, "Bin")) {
```

Comment:
You should use `std::strcmp` and include `<cstring>` header instead of `<string.h>`

Reply:
Good catch! that one had actually bothered me too :)